spec/factories/itms_factories.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :itms_intensity, class: Itms::Intensity do
+    itms_id { 1 }
+    itms_href { '/v2/intenzita/1' }
+    nazov { 'Intenzita 312020/7170/LDR/30794536' }
+    deleted_at { }
```

Comment:
tu by som explicitne nastavil `nil`

```suggestion
    deleted_at { nil }
```